README.md

```diff
@@ -75,7 +83,8 @@ Build and start the Docker image for the Solr server:
 
 Open up another shell and create the search index:
 
-    docker-compose run --rm app ./manage.py make_search_index
+    docker-compose run --rm app ./manage.py update_index
```

Comment:
you do not have to do address this in this PR, but we should use the makefile commands here, imo.

Reply:
I'm not sure what the application of this Makefile is, to be honest. There's one reference to the `import_google_docs` target in the README, but I don't see other references to the recipes in the code base (including deployment scripts). I think I'd like to update the Makefile in our import refactor so it has some useful targets for local development and eventual deployment on Heroku. I'll plan to make those changes in a separate PR.